db/seeds.rb

```diff
@@ -34,12 +34,12 @@
-@doctor3 = Doctor.create!(user: @user1, name: 'Ali Abbani', location: 'Lebanon', specification: 'Plastic surgery', picture: 'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png', price: '300', description: 'friendly and caring')
-@doctor4 = Doctor.create!(user: @user1, name: 'Henry Eze', location: 'Nigeria', specification: 'Dentist', picture: 'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png', price: '25', description: 'friendly and caring')
-@doctor5 = Doctor.create!(user: @user1, name: 'William Pedro', location: 'Brazil', specification: 'Eye specialist1', picture: 'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png', price: '80', description: 'friendly and caring')
-@doctor10 = Doctor.create!(user: @user10, name: 'xi', location: 'Turkey', specification: 'Eye specialist', picture: 'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png', price: '100', description: 'friendly and caring')
+@doctor1 = Doctor.create!(user: @user1, name: 'Ahmed Bozaci', location: 'Turkey', specification: 'Eye specialist', picture: 'https://media.istockphoto.com/photos/portrait-of-male-doctor-in-white-coat-and-stethoscope-standing-in-picture-id1327024466?k=20&m=1327024466&s=612x612&w=0&h=TDbIUqY-iPy1EQnOJvXgQKrTMrEAH0ekqNoWY8jei1g=', price: '$100', description: 'active listener and ask open-ended questions')
+@doctor2 = Doctor.create!(user: @user1, name: 'Tufoin Nkuo', location: 'Cameroon', specification: 'Heart specialist', picture: 'https://media.istockphoto.com/photos/doctors-tools-get-more-advanced-by-the-day-picture-id1300516764?k=20&m=1300516764&s=612x612&w=0&h=xhT-plNpcmdl96Vmu_Dv1ZZiEn9IW-Ti9b0UvbUnapk=', price: '$180', description: 'Honest but also offer hope, even when a situation is difficult')
+@doctor3 = Doctor.create!(user: @user1, name: 'Ali Abbani', location: 'Lebanon', specification: 'Plastic surgery', picture: 'https://cdn.pixabay.com/photo/2021/02/09/06/44/doctor-5997500__340.jpg', price: '$95', description: 'willing to go the extra mile for his patients')
+@doctor4 = Doctor.create!(user: @user1, name: 'Henry Eze', location: 'Nigeria', specification: 'Dentist', picture: 'https://media.istockphoto.com/photos/if-you-need-me-just-call-me-picture-id1288813313?k=20&m=1288813313&s=612x612&w=0&h=NiAjImsLoy5_ssvIjtMdXmG5Hqmm8uYdIpNhfqfuvHE=', price: '$80', description: 'do whatever it takes to help meet his patients’ needs')
+@doctor5 = Doctor.create!(user: @user1, name: 'William Pedro', location: 'Brazil', specification: 'geneticist ', picture: 'https://image.shutterstock.com/image-photo/cheerful-mature-doctor-posing-smiling-260nw-1384243295.jpg', price: '$110', description: 'friendly and caring')
```

Comment:
Please make sure to remove the $ sign from the price. This is affecting our seed file. We can add the $ sign in the front end.